sniffr/dog_routes/dog_routes.py

```diff
@@ -31,37 +34,39 @@ def post_dog():
 
     # If dog_id not in body then they are trying to create
     # If dog_id in body then updating content
-    if content["dog_id"]:
-        # update dog
-        ...
+    if "dog_id" in content.keys():
+        queried_dog = db.session.query(Dog).filter_by(dog_id=content['dog_id']).all()
+        if queried_dog:
+            # update dog
+            payload = jsonify(
+                {"Message": f"Successfully pinged API but editing dog id #{content['dog_id']} is not available yet."}
+            )
+            response = make_response(jsonify({"Message": f"Successfully pinged API but editing dog id #{content['dog_id']} is not available yet."}), 200)
+            response.headers["Content-Type"] = "application/json"
+            return response
+        
+        else:
+            payload = jsonify({"Message": "Dog Not Found"})
+            response = make_response(payload, 400)
+            response.headers["Content-Type"] = "application/json"
+            return response
 
     else:
         # create dog
         dog_name = content["dog_name"]
         user_id = content["user_id"]
-        breed_id = content["breed_id"]
-        size_id = content["size_id"]
-        temperament_id = content["temperament_id"]
         age = content["age"]
         sex = content["sex"]
-        is_vaccinated = content["is_vaccinated"]
-        is_fixed = content["is_fixed"]
 
```

Comment:
What's the reason for getting rid of `breed_id`, `size_id`, `temperament_id`, `is_vaccinated`, and `is_fixed`? Is this just temporary?

Reply:
Just temporary plus other people are creating the `breed`, `size`, and `temperament` tables. I am more just debugging the dog routes than completing them.